src/bosh_azure_cpi/spec/unit/vm_manager/create/vm_is_not_created_spec.rb

```diff
@@ -45,7 +45,11 @@
           end
 
           before do
-            vm_props.availability_set = availability_set_name
+            vm_props.availability_set = Bosh::AzureCloud::AvailabilitySetConfig.new(
```

Comment:
It seems that `attr_writer` for availability_set is only for tests. Can we use other ways here so that we can remove `attr_writer`?

Reply:
no very very good way to do it.

Reply:
Okay. Mark it pending.